responsibleai_vision/responsibleai_vision/rai_vision_insights/rai_vision_insights.py

```diff
@@ -738,19 +738,26 @@ def _generate_od_error_labels(self, true_y, pred_y, class_names):
                 else:
                     image_labels[_INCORRECT][object_label] += 1
 
-                image_labels[_INCORRECT][object_label] += \
-                    np.count_nonzero(
-                        error_matrix[label_idx] ==
-                        ErrorLabelType.DUPLICATE_DETECTION)
+                duplicate_detections = np.count_nonzero(
+                    error_matrix[label_idx] ==
+                    ErrorLabelType.DUPLICATE_DETECTION)
+                if duplicate_detections > 0:
+                    image_labels[_INCORRECT][object_label] += \
+                        duplicate_detections
+
+            correct_labels = sorted(image_labels[_CORRECT].items(),
+                                    key=lambda x: class_names.index(x[0]))
+            incorrect_labels = sorted(image_labels[_INCORRECT].items(),
+                                      key=lambda x: class_names.index(x[0]))
```

Comment:
Surprised our linter accepts this - I recall in some of our other repos lambdas were not allowed.  I think it's fine to merge though.